extra/restart-datadog-agent.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+if [[ $DISABLE_DATADOG_AGENT ]]; then
+  echo "DISABLE_DATADOG_AGENT environment variable is set, not starting the agent."
+  exit 0
+fi
+
+# Update integration directory configuration
+sed -i -e"s/^.*UNIX_CONFIG_PATH =.*$/UNIX_CONFIG_PATH = '\/app\/.apt\/opt\/datadog-agent\/agent'/" /app/.apt/opt/datadog-agent/agent/config.py
```

Comment:
I might be missing something, I think the datadog agent is looking for `/etc/dd-agent` and we need to  change  the file to look for `/app/.apt/etc/dd-agent/conf.d`